content/docs/use-cases/experiment-tracking.md

```diff
@@ -0,0 +1,99 @@
+│ ├── 9c32227 [exp-17dd9] │ 04:46 PM     │ 0.23687 │ 0.9167 │ 10           │ 256              │
+│ ├── 8a9cb15 [exp-29d93] │ 04:46 PM     │ 0.24459 │ 0.9134 │ 10           │ 128              │
+└─────────────────────────┴──────────────┴─────────┴────────┴──────────────┴──────────────────┘
+```
+
+![]() _Organized experiments in DVC_
+
+> 📖 Ready to dive in? See [Get Started: Experiments](/doc/start/experiments).
+
+Unlike tools that focus on experiment navigation, DVC's approach guarantees
+reproducibility by working on top of Git, instead of on being a side system with
+fragile links to Git. This means that your [data is
+tracked][versioning everything] from within project versions (without storing it
+in Git). It also enables distributed collaboration along the way, via optional
+hosting like GitHub or GitLab.
+
+DVC projects provide a seamless, local-first experience (no need for running
+servers, special services, or loading websites). You control where and how your
+data is saved and shared. DVC also improves storage efficiency via
+<abbr>caching</abbr>, preventing repetitive data transfers on every experiment.
+
+On top of all that, DVC is completely language agnostic. You can expect the same
+convenience and performance whether you're using Jupyter Notebooks or Scala, CSV
+data frames or HDFS.
+
+|                   | DVC Experiments        | MLFlow                  | Weights & Biases     | TensorBoard     |
+| ----------------- | ---------------------- | ----------------------- | -------------------- | --------------- |
+| UI                | Terminal               | Web (local or hosted)   | Web (hosted)         | Web (hosted)    |
+| Infrastructure    | None (just Git)        | Web server              | SaaS                 | SaaS            |
+| Versioning        | Git (standard, robust) | Proprietary metadata    | Proprietary metadata | None            |
+| Language Support  | Any<br/>(agnostic)     | Several (REST)          | Python only          | Python only     |
+| Compute & Storage | User (yours)           | User (yours)            | Hosted (theirs)      | Hosted (theirs) |
+| Licensing         | Open Source            | Open Source             | Proprietary          | Open Source     |
+| Cost              | Free + [storage]       | Free + [cloud][dbricks] | [$$$][wandb]         | Free (limited)  |
+
+[storage]: /doc/command-reference/remote/add#supported-storage-types
+[dbricks]: https://databricks.com/product/pricing
+[wandb]: https://wandb.ai/site/pricing
+
+> 💡 Note that other experiment tracking tools can be complementary with DVC,
+> for example for more detailed experiment logging with specialized analytics
+> and visualizations.
+
+To complete your stack, we also offer [DVC Studio], an advanced web interface to
+collaborate on DVC projects online, including full experiment management
+support. You can also integrate with CI/CD using [CML], and run experiments
+straight from Studio!
```

Comment:
Perhaps add links to:

- https://dvc.org/doc/use-cases/ci-cd-for-machine-learning 
- https://dvc.org/doc/studio/user-guide/run-experiments

Reply:
Perhaps but it may be link overflow.